pkg/stack/unwind/unwind_table.go

```diff
@@ -200,37 +203,84 @@ func (ptb *UnwindTableBuilder) PrintTable(writer io.Writer, file *objectfile.Obj
 }
 
 func ReadFDEs(file *objectfile.ObjectFile) (frame.FrameDescriptionEntries, elf.Machine, error) {
+	// Try the normal path, if it fails try the alternatives and go with them if they
```

Comment:
It worth checking our debuginfo package. Especially the `Find`. I believe we can reuse some code pieces.

Reply:
Thanks for the tip!  I especially want to nail the CRC checking before landing this.